Lux/lux_react/src/components/panes/AudioPane.jsx

```diff
@@ -0,0 +1,302 @@
+import React, { useState, useCallback, useEffect } from 'react';
```

Comment:
Remove live audio frequency bands and just use mixer with four channels. Make mixer channels narrower so all four can fit side by side on mobile. Use vertical slider for gain instead of horizontal.